exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -0,0 +1,1365 @@
+			RemainingAmount: orderDetail.Size - orderDetail.DealSize,
+			Amount:          orderDetail.Size,
+			Price:           orderDetail.Price,
+			Date:            orderDetail.CreatedAt.Time(),
+		}, nil
+	default:
+		return order.Detail{}, fmt.Errorf("%w %v", asset.ErrNotSupported, assetType)
+	}
+}
+
+// GetDepositAddress returns a deposit address for a specified currency
+func (ku *Kucoin) GetDepositAddress(ctx context.Context, c currency.Code, accountID, chain string) (*deposit.Address, error) {
+	ad, err := ku.GetDepositAddressesV2(ctx, c.Upper().String())
+	if err != nil {
+		fad, err := ku.GetFuturesDepositAddress(ctx, c.String())
+		if err != nil {
+			return nil, err
+		}
+		return &deposit.Address{
+			Address: fad.Address,
+			Chain:   fad.Chain,
+			Tag:     fad.Memo,
+		}, nil
+	}
+	if len(ad) > 1 {
+		return nil, errMultipleDepositAddress
+	} else if len(ad) == 0 {
+		return nil, errNoDepositAddress
+	}
+	return &deposit.Address{
+		Address: ad[0].Address,
+		Chain:   ad[0].Chain,
+		Tag:     ad[0].Memo,
+	}, nil
+}
+
+// WithdrawCryptocurrencyFunds returns a withdrawal ID when a withdrawal is
+// submitted
+// The endpoint was deprecated for futures, please transfer assets from the FUTURES account to the MAIN account first, and then withdraw from the MAIN account
+func (ku *Kucoin) WithdrawCryptocurrencyFunds(ctx context.Context, withdrawRequest *withdraw.Request) (*withdraw.ExchangeResponse, error) {
+	if err := withdrawRequest.Validate(); err != nil {
+		return nil, err
+	}
+	withdrawalID, err := ku.ApplyWithdrawal(ctx, withdrawRequest.Currency.Upper().String(), withdrawRequest.Crypto.Address, withdrawRequest.Crypto.AddressTag, withdrawRequest.Description, withdrawRequest.Crypto.Chain, "INTERNAL", false, withdrawRequest.Amount)
+	if err != nil {
+		return nil, err
+	}
+	return &withdraw.ExchangeResponse{
+		ID: withdrawalID,
+	}, nil
+}
+
+// WithdrawFiatFunds returns a withdrawal ID when a withdrawal is submitted
+func (ku *Kucoin) WithdrawFiatFunds(ctx context.Context, withdrawRequest *withdraw.Request) (*withdraw.ExchangeResponse, error) {
+	return nil, common.ErrFunctionNotSupported
+}
+
+// WithdrawFiatFundsToInternationalBank returns a withdrawal ID when a withdrawal is
+// submitted
+func (ku *Kucoin) WithdrawFiatFundsToInternationalBank(ctx context.Context, withdrawRequest *withdraw.Request) (*withdraw.ExchangeResponse, error) {
+	return nil, common.ErrFunctionNotSupported
+}
+
+// GetActiveOrders retrieves any orders that are active/open
+func (ku *Kucoin) GetActiveOrders(ctx context.Context, getOrdersRequest *order.GetOrdersRequest) (order.FilteredOrders, error) {
+	err := ku.CurrencyPairs.IsAssetEnabled(getOrdersRequest.AssetType)
+	if err != nil {
+		return nil, err
+	}
+	if getOrdersRequest.Validate() != nil {
+		return nil, err
+	}
+	format, err := ku.GetPairFormat(getOrdersRequest.AssetType, true)
+	if err != nil {
+		return nil, err
+	}
+	pair := ""
+	orders := []order.Detail{}
+	switch getOrdersRequest.AssetType {
```

Comment:
Doesn't look like Any is being addressed still:
[ERROR] | ORDER | 20/02/2023 14:50:21 | Unable to get active orders for Kucoin and asset type spot: code: 400100 message: validation.sideInvalid

Reply:
`side` is an optional param. I made the previous comment because the GoCryptoTrader order manager retrieves orders with the asset type of `Any` to retreive all possible active orders which I detailed [here](https://github.com/thrasher-corp/gocryptotrader/pull/1102#discussion_r1103971824). The function you have created will return another error. You can test this by running the application and seeing the error
`[ERROR] | ORDER | 21/02/2023 16:24:01 | Unable to get active orders for Kucoin and asset type spot: unsupported order side`